examples/08_jotai-demo/src/lib/waku-jotai/client.tsx

```diff
@@ -80,6 +80,7 @@ export const BaseSyncAtoms = ({
         const rscParams = {
           jotai_atomValues: serializedAtomValues,
         };
+        // eslint-disable-next-line @typescript-eslint/no-floating-promises
         refetch(...createRscPathAndRscParams(rscPath, rscParams));
```

Comment:
```suggestion
        void refetch(...createRscPathAndRscParams(rscPath, rscParams));
```

Reply:
This is intentional. To me, `void` implies "it's good." No, it's not good. We shouldn't hide errors (or they will be uncaught). So, in other places, I added `.catch`. For `waku-jotai`, I'll revisit and `eslint-disable` shows less "it's good."